src/client/src/components/pages/courses/addCourses/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import {
+  state as initialState,
+  fields as fieldSet,
+  validationForm
+} from "./staticData";
+import Form from "./../../../abstract/Form";
+import Footer from '../../../abstract/footer';
+
+export default class index extends Component {
+  state = initialState;
+
+  onChange = event => {
+    const { value, name } = event.target;
+    this.setState({ [name]: value });
+  };
+
+  clearFields = event => {
+    event.preventDefault();
+    const fields = this.state;
+    for (const key in fields) {
+      fields[key] = "";
+    }
+    this.setState(fields);
+  };
+
+  // the implemention waiting  back end api
+  onSubmit = event => {
+    event.preventDefault();
+    console.log(event.target);
+    const fields = { ...this.state };
+    const error = validationForm(fields);
+    if (error) return this.setState({ error });
+
+    for (const key in fields) {
+      fields[key] = "";
+    }
+    this.setState(fields);
+  };
+
+  render() {
+    return (
+      <div>
+        <Form
+          title="Add Manager"
```

Comment:
It should be "Add course" instead of "Add manager"

Reply:
done